h/static/scripts/annotator/guest.coffee

```diff
@@ -1,7 +1,20 @@
-Promise = require('es6-promise').Promise
+raf = require('raf')
+Promise = global.Promise ? require('es6-promise').Promise
```

Comment:
Oh, I totally understand now. I should be just getting `Promise` from the global and doing `require('es6-promise/polyfill')`. The entire problem is because I'm requiring the polyfill version directly. Thanks for catching that.